scripts/helper.js

```diff
@@ -659,6 +659,9 @@ var permissions = [{
 }, { 
   target: `${accounts.proposals.account}@active`,
   actor: `${accounts.escrow.account}@active`
+}, {
```

Comment:
Maybe as a matter of best practices, can we limit these to specific actions?

I don't like to add broad generic rights like this - but there's a way to add rights for single actions, which we've used elsewhere. And we actually don't need a separate key for this either, it's just a permission that allows code to execute a specific action, then the contract - in this case history - gets the right to that permission. 

I think we will have to change all the generic ones to specific ones at some point.

Reply:
Ok, it sounds to be better to have specific permissions, I changed it.